model.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+
+class Model {
+    constructor(data) {
+        this.data = data;
+    }
+
+    static read() {
+        let data = fs.readFileSync('./data.json', 'utf8');
+        data = JSON.parse(data);
+        return data;
```

Comment:
setelah melakukan JSON.parse, data sebaiknya diubah dari array of object menjadi array of instance model, baru setelah itu di-return ke controller